compaction/compaction_manager.cc

```diff
@@ -1144,6 +1146,7 @@ class compaction_manager::rewrite_sstables_compaction_task : public compaction_m
             const auto& res = co_await rewrite_sstable(std::move(sst));
             cts.validation_errors += res.validation_errors;
         }
+        _cm.validation_errors += cts.validation_errors;   // TODO Deexie  maybe it should be done after every rewrite_sstable to provide gradual growth (?)
```

Comment:
Is it ok if `_cm.validation_errors` is updated here and not in every loop iteration?